pkg/apis/admissionregistration/v1beta1/defaults.go

```diff
@@ -97,14 +97,6 @@ func SetDefaults_MutatingWebhook(obj *admissionregistrationv1beta1.MutatingWebho
 	}
 }
 
-// SetDefaults_Rule sets defaults for webhook rule
```

Comment:
this is safe because v1 defaulting matches v1beta1 defaulting (verified generated defaults switched to calling v1.SetDefaults_Rule, which is identical)